grumphp.yml

```diff
@@ -3,8 +3,4 @@ parameters:
   bin_dir: "./vendor/bin"
   git_dir: "."
   tasks:
-      phpcs:
-        standard: PSR2
-        ignore_patterns:
-          - "*Spec.php"
-      phpspec: ~
+      codeception: ~
```

Comment:
Actually, I think this change is wrong and is causing the Travis build to fail - GrumPHP doesn't have any codeception tests, we do however want to be PSR2 compliant and run our specs. :)